src/stories/plans/stepsTable.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+<Meta title="Plans/StepsTable/Docs" />
+
+# StepsTable
+
+**Shows detailed information about the Steps of an installation.**
+
+## Overview
+
+The **StepsTable** component is an
+[SLDS Data Table](https://react.lightningdesignsystem.com/components/data-tables/)
+used to display information about each **Step** involved in installing a
+**Plan**.
+
+If a pre-install validation check is required, the **StepsTable** initially only
+displays **Step** names and types.
+
+<Canvas>
+  <Story id="plans-stepstable-examples--pre-validation" />
+</Canvas>
+
+Once the installation process can be started an `Install` column appears. If a
+**Step** is required, the `Install` column for that **Step** features a
+checkmark. If it is optional, a selectable checkbox is shown instead.
+
+<Canvas>
+  <Story id="plans-stepstable-examples--post-validation" />
+</Canvas>
+
+During the installation process for a **Step**, a toggle button appears next to
+its name. Clicking on the button toggles an expanded log of information about
+the time, status, and detailed processes of the **Step**.
+
+When an installation **Step** is complete, a round, green icon with a white
+checkmark appears in the `Install` column.
+
+A toggle button for hiding or expanding logs for the active step (or all steps,
+if the installation is not running) appears in the `Steps` column header once
+the first **Step** starts to transmit log data.
+
+Skipped **Steps** display a gray dash in the `Install` column. If a **Step**
+fails its status is labeled as `Error` and an error message is shown below the
+**Step** name.
+
```

Comment:
@dvdherron maybe since we don't show this (I think you mentioned you did but @jgerigmeyer thought it would be best to show the non-error states on the landing page) we could add a relative link to the error example perhaps?
 `/path=/story/plans-stepstable-examples--post-validation&args=preflight:Error`

Reply:
> 
> 
> @dvdherron maybe since we don't show this (I think you mentioned you did but @jgerigmeyer thought it would be best to show the non-error states on the landing page) we could add a relative link to the error example perhaps?
> `/path=/story/plans-stepstable-examples--post-validation&args=preflight:Error`


I tried this @stacyk , but since that addition at the end (after `--post-validation`) doesn't match any of the id's the component doesn't render correctly. I think we would have to make a separate "Failed" example to render it inline like this.

Reply:
@dvdherron Ah, yes, I see. I think it is fine as is. If it is unclear they can jump to the component and play with the controls :)